CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added new ShowMore component
```

Comment:
Can we change the name of this if it's going to be a separate component? `SummaryButton` ?

Reply:
Initially it was meant to be solely for 'show more' functionality, but I guess this could work as more generic 'Summary'.